saleor/checkout/views/summary.py

```diff
@@ -20,7 +21,8 @@ def create_order(checkout):
         return None, redirect('checkout:summary')
     checkout.clear_storage()
     checkout.cart.clear()
-    order.create_history_entry()
+    order.create_history_entry(status=OrderStatus.NEW, comment=pgettext_lazy(
```

Comment:
Shouldn't it also set the user who placed the order?

Reply:
Yes, added